device/esp_tinyusb/usb_descriptors.c

```diff
@@ -182,7 +182,7 @@ uint8_t const descriptor_cfg_kconfig[] = {
 
 #if CFG_TUD_MSC
     // Interface number, string index, EP Out & EP In address, EP size
-    TUD_MSC_DESCRIPTOR(ITF_NUM_MSC, STRID_MSC_INTERFACE, EPNUM_MSC, 0x80 | EPNUM_MSC, 64), // highspeed 512
+    TUD_MSC_DESCRIPTOR(ITF_NUM_MSC, STRID_MSC_INTERFACE, EPNUM_MSC, 0x80 | EPNUM_MSC, TUD_OPT_HIGH_SPEED ? 512 : 64),
```

Comment:
Could we do the same for default NCM descriptor? This way we won't hit the same problem once we port HighSpeed NCM in the future

Reply:
@tore-espressif For NCM class default configuration descriptor has already the correct size of BULK EP: 
`#define CFG_TUD_NET_ENDPOINT_SIZE (TUD_OPT_HIGH_SPEED ? 512 : 64)`
For INTR EP I am not sure that this is important, because the INTR could be any size in HS/FS, not strictly 512/64....
But let me check that additionally.

Reply:
Thank you! I indeed look at the INTR EP size, which is 64 and correct.